x86_64/dumux/cactus.yaml

```diff
@@ -0,0 +1,10 @@
+nvchecker:
+  - source: aur
+    aur:
```

Comment:
Seems that the PKGBUILDs of all these package will also build a python binding package. I'm afraid that they all need the python upstream.

smart add package script has the -t option to change the default template.